styles/syntax/elixir.less

```diff
@@ -0,0 +1,26 @@
+.syntax--source.syntax--elixir {
+  .syntax--punctuation.definition.constant {
```

Comment:
All the classes need the `syntax--` prefix, not just the first of the "chain". So the ☝️ above line would need to be changed to:

```less
.syntax--punctuation.syntax--definition.syntax--constant {
```

Same for the rest below👇 .